packages/docs/pages/usage/search/sort.mdx

```diff
@@ -0,0 +1,58 @@
+# Sort
+
+To sort, Orama needs to know on which properties you want to sort.
```

Comment:
Is this still needed? Given that we can apply sorting on every property (except for booleans), we might not need a `sortSchema`